Add `report-job-status` action (#9)
tarantool/actions: report-job-status/README.md

# GitHub Action: report failed job to VK Team chat

> This action composes a message about the failed job and sends it to VK Teams chat.

Reviewer: ```suggestion

This action composes a message about a failed job and sends it to VK Teams chat.
```

Author: done

> The message contains the following information:

Reviewer: ```suggestion
The message contains the following information:

```

Author: done

* __Job__: name of the failed job with the link to this job

> * __Commit__: hash of the commit started the job with the link to this commit

Reviewer: ```suggestion
* __Commit__: hash of the commit that triggered the job with the link to this commit
```

* __Branch__: name of the branch with the link to this branch
* __History__: the link to commit history

> * __Triggered on__: name of the event triggered the job 

Author: done

* __Committer__: GitHub login of the commit author

> * __Commit message subject__: subject (first string) of the commit message

Author: done

* __Failed steps info__: info about the failed steps (outputs, outcome, conclusion)

> # Prepare your workflow
> The action can tell you about the exact steps that failed only if your steps have IDs, example:
> ```yaml

Reviewer: ```suggestion
# Prepare your workflow

The action can tell you about the exact steps that failed only if your steps have IDs, for example:

```yaml
```

Author: done

steps:
  - name: This step does some useful work
    id: some-useful-work
    run: some useful work

  - name: This step does some other useful work
    id: other-useful-work
    uses: some/additional/action@v1.0
    run: other useful work
```

# Usage
All the fields are required.
```yaml
  - name: Send VK Teams notification
    if: failure()
    uses: tarantool/actions/report-job-status@master
    with:
      api-url: ${{ secrets.VKTEAMS_BOT_API }}
      bot-token: ${{ secrets.VKTEAMS_BOT_TOKEN }}
      chat-id: ${{ secrets.VKTEAMS_CHAT_ID }}
      job-steps: ${{ ToJson(steps) }}
```
| variable  | how to get                                    | example                              |
|-----------|-----------------------------------------------|--------------------------------------|
| api-url   | `/help` in Metabot                            | https://myteam.mail.ru/bot/v1        |
| bot-token | API token received from Metabot               | 000.1234567890.0987654321:1111111111 |                            
| chat-id   | Can be found in the chat info                 | @tntcore_ghaction_chat               |

> | job-steps | Please, set exactly as `${{ ToJson(steps) }}` | ${{ ToJson(steps) }}                 |

Reviewer: ```suggestion
| job-steps | Must be `${{ ToJson(steps) }}` | ${{ ToJson(steps) }}                 |
```

Author: done
